src/Orchestration/Orchestration.php

```diff
@@ -128,6 +129,91 @@ public function networkConnect(string $container, string $network): bool
         return $this->adapter->networkConnect($container, $network);
     }
 
+    /**
+     * Get usage stats of containers
```

Comment:
Container or containers?

Reply:
Containers I think.

If you provide container ID, you get array with 1 Stat instance.

If you don't provide ID, you get array with Stat instance for each container that exists.